packages/node_modules/@ciscospark/react-container-read-receipts/src/selectors.js

```diff
@@ -1,14 +1,36 @@
+/* eslint-disable max-params */
 import {createSelector} from 'reselect';
 
 const getActivities = (state) => state.conversation.get(`activities`);
 const getParticipants = (state) => state.conversation.get(`participants`);
+const getCurrentUser = (state) => state.user.get(`currentUser`);
```

Comment:
I was monitoring the selector being fired and actually the opposite was true. It fired WAY more often without getting the current user.